pkg/rpcmiddleware/rpclog/client_interceptors.go

```diff
@@ -33,6 +34,9 @@ func UnaryClientInterceptor(ctx context.Context, opts ...Option) grpc.UnaryClien
 
 		startTime := time.Now()
 		err := invoker(newCtx, method, req, reply, cc, opts...)
+		if errors.IsResourceExhausted(err) {
```

Comment:
Please extract this to an unexposed function in `rpclog` to return whether to rate limit logs.